frontend/src/state/models/schedule-data/shift-info.model.ts

```diff
@@ -10,9 +8,10 @@ export enum ShiftCode {
   RPN = "RPN",
   W = "W",
   U = "U",
-  L4 = "L4"
+  L4 = "L4",
+  Wildcard = "*",
```

Comment:
zmienić nazwę na coś bardziej znaczącego - zablokowane, czy coś takiego (blocked/unpreferred?)

Reply:
Z tego co kojarzę gwiazdka nie oznacza, że zmiana jest zablokowana, więc `blocked` nie nazywałbym ją `blocked`.

Co do `unpreferred` to też jest dziwne, bo my nigdzie tego nie uwzgłędniamy: ni po stronie algorytmu ni po stronie `backendu`.

 Moim zdaniem najbardziej sensownym rozwiązaniem będzie usunąć tą zmienną

Reply:
No Paweł wspominał, że to może w przyszłości coś znaczyć, ale jak to na razie nie działa, to poprostu każdy nieznany znak będzie pustym polem

Reply:
Moja koncepcja jest taka, żeby ten parser tak łatwo się nie wywalał. Jeśli napotka coś czego nie potrafi sparsować niech traktuje to jako wolne. Przydałby się warrnig, że były napotkane błędy podczas pardowania. Póki co gwiazdka zalicza się do śmieci w Excelu.

Reply:
@prenc  Masz rację co do dodania warningu i teraz to zaimplementuję. 
@maciekb05 
Każdy nieznany znak na razie jest wolną zmianą. 
Nie wiem czy ma sens dodawać teraz obsługę symboli których znaczenia nie jesteśmy pewni

Reply:
Dodałem warningi